### PR TITLE
transfer is done

### PR DESCRIPTION
**Pull request by ab235**

### CODE REVIEW

**Review by ericz9002 — CHANGES_REQUESTED** [review]
replace flash formatting in html with 
<p class="flash flash-{{flash | getFlashType}}">{{flash | getFlashMessage}}</p>
and add "e" or "s" as first character in the flash message in addFlash to make it consistent with other code

**Review by ericz9002 — COMMENTED** [review]
do you need to include style2.css to format the flash messages to that color?

**Comment by ericz9002** [comment]
Also a flash message for entering an invalid recipient account would be nice

File: main.go [hunk 1]
```diff
@@ -485,23 +485,23 @@ func transfer(w http.ResponseWriter, r *http.Request){
 				destinationAccount,
 			).Scan(&dbal)
 			if err != nil {
-				return fmt.Errorf("Destination Account does not exist.")
+				return fmt.Errorf("eDestination Account does not exist.")
```
Comment (SharpBit) [inline main.go:488]: only AddFlash needs this e, fmt.Errorf does not

File: main.go [hunk 2]
```diff
@@ -485,23 +485,23 @@ func transfer(w http.ResponseWriter, r *http.Request){
...
 			}
 			err = conn.QueryRow(
 				context.Background(),
 				`UPDATE accounts SET balance = $1 WHERE account_num = $2 RETURNING balance`,
 				currb-amount, sourceAccount,
 			).Scan(&tmp)
 			if err != nil {
-				return fmt.Errorf("Bad Update: %v", err)
+				return fmt.Errorf("eBad Update: %v", err)
```
Comment (SharpBit) [inline main.go:496]: here too

File: main.go [hunk 3]
```diff
@@ -485,23 +485,23 @@ func transfer(w http.ResponseWriter, r *http.Request){
...
 			}
 			err = conn.QueryRow(
 				context.Background(),
 				`UPDATE accounts SET balance = $1 WHERE account_num = $2 RETURNING balance`,
 				dbal+amount, destinationAccount,
 			).Scan(&tmp)
 			if err != nil {
-				return fmt.Errorf("Bad Update: %v", err)
+				return fmt.Errorf("eBad Update: %v", err)
```
Comment (SharpBit) [inline main.go:504]: here too

File: main.go [hunk 4]
```diff
@@ -521,6 +521,9 @@ func transfer(w http.ResponseWriter, r *http.Request){
 			}
 			return nil
 		})
+		if err != nil{
+			AddFlash(r, w, err.Error())
```
Comment (SharpBit) [inline main.go:525]: Do `"e"+err.Error()` here

File: main.go [hunk 5]
```diff
@@ -521,6 +521,9 @@ func transfer(w http.ResponseWriter, r *http.Request){
...
+		}
 		http.Redirect(w, r, "/transfer", http.StatusSeeOther)
```
Comment (SharpBit) [inline main.go:527]: I would add a success flash here if there was no error

**Review by SharpBit — CHANGES_REQUESTED** [review]

**Review by ericz9002 — APPROVED** [review]

**Pull request merged**